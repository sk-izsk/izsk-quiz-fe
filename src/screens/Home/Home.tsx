import React from 'react';

export interface HomeProps {}

const Home: React.FC<HomeProps> = () => {
  return <div>hello</div>;
};

export default Home;
