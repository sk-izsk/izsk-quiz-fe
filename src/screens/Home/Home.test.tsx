import { render } from '@testing-library/react';
import React from 'react';
import Home from './Home';

test('Home renders correctly', () => {
  const { asFragment } = render(<Home />);
  expect(asFragment()).toMatchSnapshot();
});
