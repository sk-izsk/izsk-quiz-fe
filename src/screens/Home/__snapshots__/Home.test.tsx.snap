// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root MuiBox-root-5 makeStyles-mainContainer-1"
  >
    <div
      class="MuiBox-root MuiBox-root-9 makeStyles-cardContainer-6"
    >
      <div
        class="_28vIk _2bGnb _1SFBz _12WTq makeStyles-card-20"
      >
        <div
          class="_2So3v _2uw1B "
        >
          <h5
            class="_3yPwv _2TwOi _11JjB _1uvOH "
          >
            Quiz Title
          </h5>
          <p
            class="_3yPwv _2vNlX _1QRSN _1uvOH makeStyles-dateContainer-8"
          >
            date of quiz
          </p>
          <p
            class="_3yPwv _i_J81 _11JjB _1uvOH "
          >
            Result of quiz
            <br />
            Passed
          </p>
        </div>
        <div
          class="_qCt7a _1l5Ig "
        >
          <div
            class="_cL4RU _17_OS  _3W7Om _CWiZp _39_ND "
            style="color: rgb(238, 252, 243); background-color: rgb(33, 150, 243);"
          >
            <button
              class="_3lLt8"
            >
              More details
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-21 makeStyles-cardContainer-6"
    >
      <div
        class="_28vIk _2bGnb _1SFBz _12WTq makeStyles-card-20"
      >
        <div
          class="_2So3v _2uw1B "
        >
          <h5
            class="_3yPwv _2TwOi _11JjB _1uvOH "
          >
            Quiz Title
          </h5>
          <p
            class="_3yPwv _2vNlX _1QRSN _1uvOH makeStyles-dateContainer-8"
          >
            date of quiz
          </p>
          <p
            class="_3yPwv _i_J81 _11JjB _1uvOH "
          >
            Result of quiz
            <br />
            Passed
          </p>
        </div>
        <div
          class="_qCt7a _1l5Ig "
        >
          <div
            class="_cL4RU _17_OS  _3W7Om _CWiZp _39_ND "
            style="color: rgb(238, 252, 243); background-color: rgb(33, 150, 243);"
          >
            <button
              class="_3lLt8"
            >
              More details
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-22 makeStyles-cardContainer-6"
    >
      <div
        class="_28vIk _2bGnb _1SFBz _12WTq makeStyles-card-20"
      >
        <div
          class="_2So3v _2uw1B "
        >
          <h5
            class="_3yPwv _2TwOi _11JjB _1uvOH "
          >
            Quiz Title
          </h5>
          <p
            class="_3yPwv _2vNlX _1QRSN _1uvOH makeStyles-dateContainer-8"
          >
            date of quiz
          </p>
          <p
            class="_3yPwv _i_J81 _11JjB _1uvOH "
          >
            Result of quiz
            <br />
            Passed
          </p>
        </div>
        <div
          class="_qCt7a _1l5Ig "
        >
          <div
            class="_cL4RU _17_OS  _3W7Om _CWiZp _39_ND "
            style="color: rgb(238, 252, 243); background-color: rgb(33, 150, 243);"
          >
            <button
              class="_3lLt8"
            >
              More details
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-23 makeStyles-cardContainer-6"
    >
      <div
        class="_28vIk _2bGnb _1SFBz _12WTq makeStyles-card-20"
      >
        <div
          class="_2So3v _2uw1B "
        >
          <h5
            class="_3yPwv _2TwOi _11JjB _1uvOH "
          >
            Quiz Title
          </h5>
          <p
            class="_3yPwv _2vNlX _1QRSN _1uvOH makeStyles-dateContainer-8"
          >
            date of quiz
          </p>
          <p
            class="_3yPwv _i_J81 _11JjB _1uvOH "
          >
            Result of quiz
            <br />
            Passed
          </p>
        </div>
        <div
          class="_qCt7a _1l5Ig "
        >
          <div
            class="_cL4RU _17_OS  _3W7Om _CWiZp _39_ND "
            style="color: rgb(238, 252, 243); background-color: rgb(33, 150, 243);"
          >
            <button
              class="_3lLt8"
            >
              More details
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-24 makeStyles-cardContainer-6"
    >
      <div
        class="_28vIk _2bGnb _1SFBz _12WTq makeStyles-card-20"
      >
        <div
          class="_2So3v _2uw1B "
        >
          <h5
            class="_3yPwv _2TwOi _11JjB _1uvOH "
          >
            Quiz Title
          </h5>
          <p
            class="_3yPwv _2vNlX _1QRSN _1uvOH makeStyles-dateContainer-8"
          >
            date of quiz
          </p>
          <p
            class="_3yPwv _i_J81 _11JjB _1uvOH "
          >
            Result of quiz
            <br />
            Passed
          </p>
        </div>
        <div
          class="_qCt7a _1l5Ig "
        >
          <div
            class="_cL4RU _17_OS  _3W7Om _CWiZp _39_ND "
            style="color: rgb(238, 252, 243); background-color: rgb(33, 150, 243);"
          >
            <button
              class="_3lLt8"
            >
              More details
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-25 makeStyles-cardContainer-6"
    >
      <div
        class="_28vIk _2bGnb _1SFBz _12WTq makeStyles-card-20"
      >
        <div
          class="_2So3v _2uw1B "
        >
          <h5
            class="_3yPwv _2TwOi _11JjB _1uvOH "
          >
            Quiz Title
          </h5>
          <p
            class="_3yPwv _2vNlX _1QRSN _1uvOH makeStyles-dateContainer-8"
          >
            date of quiz
          </p>
          <p
            class="_3yPwv _i_J81 _11JjB _1uvOH "
          >
            Result of quiz
            <br />
            Passed
          </p>
        </div>
        <div
          class="_qCt7a _1l5Ig "
        >
          <div
            class="_cL4RU _17_OS  _3W7Om _CWiZp _39_ND "
            style="color: rgb(238, 252, 243); background-color: rgb(33, 150, 243);"
          >
            <button
              class="_3lLt8"
            >
              More details
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-26 makeStyles-cardContainer-6"
    >
      <div
        class="_28vIk _2bGnb _1SFBz _12WTq makeStyles-card-20"
      >
        <div
          class="_2So3v _2uw1B "
        >
          <h5
            class="_3yPwv _2TwOi _11JjB _1uvOH "
          >
            Quiz Title
          </h5>
          <p
            class="_3yPwv _2vNlX _1QRSN _1uvOH makeStyles-dateContainer-8"
          >
            date of quiz
          </p>
          <p
            class="_3yPwv _i_J81 _11JjB _1uvOH "
          >
            Result of quiz
            <br />
            Passed
          </p>
        </div>
        <div
          class="_qCt7a _1l5Ig "
        >
          <div
            class="_cL4RU _17_OS  _3W7Om _CWiZp _39_ND "
            style="color: rgb(238, 252, 243); background-color: rgb(33, 150, 243);"
          >
            <button
              class="_3lLt8"
            >
              More details
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-27 makeStyles-cardContainer-6"
    >
      <div
        class="_28vIk _2bGnb _1SFBz _12WTq makeStyles-card-20"
      >
        <div
          class="_2So3v _2uw1B "
        >
          <h5
            class="_3yPwv _2TwOi _11JjB _1uvOH "
          >
            Quiz Title
          </h5>
          <p
            class="_3yPwv _2vNlX _1QRSN _1uvOH makeStyles-dateContainer-8"
          >
            date of quiz
          </p>
          <p
            class="_3yPwv _i_J81 _11JjB _1uvOH "
          >
            Result of quiz
            <br />
            Passed
          </p>
        </div>
        <div
          class="_qCt7a _1l5Ig "
        >
          <div
            class="_cL4RU _17_OS  _3W7Om _CWiZp _39_ND "
            style="color: rgb(238, 252, 243); background-color: rgb(33, 150, 243);"
          >
            <button
              class="_3lLt8"
            >
              More details
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-28 makeStyles-cardContainer-6"
    >
      <div
        class="_28vIk _2bGnb _1SFBz _12WTq makeStyles-card-20"
      >
        <div
          class="_2So3v _2uw1B "
        >
          <h5
            class="_3yPwv _2TwOi _11JjB _1uvOH "
          >
            Quiz Title
          </h5>
          <p
            class="_3yPwv _2vNlX _1QRSN _1uvOH makeStyles-dateContainer-8"
          >
            date of quiz
          </p>
          <p
            class="_3yPwv _i_J81 _11JjB _1uvOH "
          >
            Result of quiz
            <br />
            Passed
          </p>
        </div>
        <div
          class="_qCt7a _1l5Ig "
        >
          <div
            class="_cL4RU _17_OS  _3W7Om _CWiZp _39_ND "
            style="color: rgb(238, 252, 243); background-color: rgb(33, 150, 243);"
          >
            <button
              class="_3lLt8"
            >
              More details
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-29 makeStyles-cardContainer-6"
    >
      <div
        class="_28vIk _2bGnb _1SFBz _12WTq makeStyles-card-20"
      >
        <div
          class="_2So3v _2uw1B "
        >
          <h5
            class="_3yPwv _2TwOi _11JjB _1uvOH "
          >
            Quiz Title
          </h5>
          <p
            class="_3yPwv _2vNlX _1QRSN _1uvOH makeStyles-dateContainer-8"
          >
            date of quiz
          </p>
          <p
            class="_3yPwv _i_J81 _11JjB _1uvOH "
          >
            Result of quiz
            <br />
            Passed
          </p>
        </div>
        <div
          class="_qCt7a _1l5Ig "
        >
          <div
            class="_cL4RU _17_OS  _3W7Om _CWiZp _39_ND "
            style="color: rgb(238, 252, 243); background-color: rgb(33, 150, 243);"
          >
            <button
              class="_3lLt8"
            >
              More details
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-30 makeStyles-cardContainer-6"
    >
      <div
        class="_28vIk _2bGnb _1SFBz _12WTq makeStyles-card-20"
      >
        <div
          class="_2So3v _2uw1B "
        >
          <h5
            class="_3yPwv _2TwOi _11JjB _1uvOH "
          >
            Quiz Title
          </h5>
          <p
            class="_3yPwv _2vNlX _1QRSN _1uvOH makeStyles-dateContainer-8"
          >
            date of quiz
          </p>
          <p
            class="_3yPwv _i_J81 _11JjB _1uvOH "
          >
            Result of quiz
            <br />
            Passed
          </p>
        </div>
        <div
          class="_qCt7a _1l5Ig "
        >
          <div
            class="_cL4RU _17_OS  _3W7Om _CWiZp _39_ND "
            style="color: rgb(238, 252, 243); background-color: rgb(33, 150, 243);"
          >
            <button
              class="_3lLt8"
            >
              More details
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-31 makeStyles-cardContainer-6"
    >
      <div
        class="_28vIk _2bGnb _1SFBz _12WTq makeStyles-card-20"
      >
        <div
          class="_2So3v _2uw1B "
        >
          <h5
            class="_3yPwv _2TwOi _11JjB _1uvOH "
          >
            Quiz Title
          </h5>
          <p
            class="_3yPwv _2vNlX _1QRSN _1uvOH makeStyles-dateContainer-8"
          >
            date of quiz
          </p>
          <p
            class="_3yPwv _i_J81 _11JjB _1uvOH "
          >
            Result of quiz
            <br />
            Passed
          </p>
        </div>
        <div
          class="_qCt7a _1l5Ig "
        >
          <div
            class="_cL4RU _17_OS  _3W7Om _CWiZp _39_ND "
            style="color: rgb(238, 252, 243); background-color: rgb(33, 150, 243);"
          >
            <button
              class="_3lLt8"
            >
              More details
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="MuiBox-root MuiBox-root-32 makeStyles-paginateContainer-3"
  >
    <nav
      aria-label="pagination navigation"
      class="MuiPagination-root"
    >
      <ul
        class="MuiPagination-ul"
      >
        <li>
          <button
            aria-label="Go to previous page"
            class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page MuiPaginationItem-outlined MuiPaginationItem-outlinedPrimary Mui-disabled Mui-disabled"
            disabled=""
            tabindex="-1"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiPaginationItem-icon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z"
              />
            </svg>
          </button>
        </li>
        <li>
          <button
            aria-current="true"
            aria-label="page 1"
            class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page MuiPaginationItem-outlined MuiPaginationItem-outlinedPrimary Mui-selected"
            tabindex="0"
            type="button"
          >
            1
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </li>
        <li>
          <button
            aria-label="Go to page 2"
            class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page MuiPaginationItem-outlined MuiPaginationItem-outlinedPrimary"
            tabindex="0"
            type="button"
          >
            2
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </li>
        <li>
          <button
            aria-label="Go to page 3"
            class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page MuiPaginationItem-outlined MuiPaginationItem-outlinedPrimary"
            tabindex="0"
            type="button"
          >
            3
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </li>
        <li>
          <button
            aria-label="Go to page 4"
            class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page MuiPaginationItem-outlined MuiPaginationItem-outlinedPrimary"
            tabindex="0"
            type="button"
          >
            4
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </li>
        <li>
          <button
            aria-label="Go to page 5"
            class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page MuiPaginationItem-outlined MuiPaginationItem-outlinedPrimary"
            tabindex="0"
            type="button"
          >
            5
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </li>
        <li>
          <div
            class="MuiPaginationItem-root MuiPaginationItem-ellipsis"
          >
            …
          </div>
        </li>
        <li>
          <button
            aria-label="Go to page 10"
            class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page MuiPaginationItem-outlined MuiPaginationItem-outlinedPrimary"
            tabindex="0"
            type="button"
          >
            10
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </li>
        <li>
          <button
            aria-label="Go to next page"
            class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page MuiPaginationItem-outlined MuiPaginationItem-outlinedPrimary"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiPaginationItem-icon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </li>
      </ul>
    </nav>
  </div>
</DocumentFragment>
`;
